staging/src/k8s.io/component-base/metrics/options.go

```diff
@@ -56,12 +57,21 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 			"The format is <major>.<minor>, e.g.: '1.16'. "+
 			"The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics, "+
 			"rather than being surprised when they are permanently removed in the release after that.")
+	fs.StringSliceVar(&o.DisabledMetrics,
+		"disabled-metrics",
+		o.DisabledMetrics,
+		"This flag provides an escape hatch for misbehaving metrics. "+
+			"You must provide the fully qualified metric name in order to disable it.")
 }
 
 // Apply applies parameters into global configuration of metrics.
 func (o *Options) Apply() {
 	if o != nil && len(o.ShowHiddenMetricsForVersion) > 0 {
```

Comment:
Should this flag work together with `show-hidden-metrics-for-version`?

Reply:
No, this is higher precedence since it is an explicit shunting off of a metric.